internal/initrec/recommenders/ruby/ruby.go

```diff
@@ -64,8 +64,6 @@ func parseRubyVersion(gemfile string) string {
 			return matches[2]
 		}
 	}
-	if err := s.Err(); err != nil {
-		return ""
-	}
-	return "" // not found
```

Comment:
I wonder why it was written this way before 😂

Reply:
do we want to do anything with the error? (this is a strict improvement so not really a blocker).